src/server/game/Entities/Creature/GossipDef.h

```diff
@@ -237,6 +234,21 @@ class TC_GAME_API QuestMenu
         QuestMenuItemList _questMenuItems;
 };
 
+class SelectionData
+{
+    public:
+        SelectionData() { Reset(); }
+
+        void Reset()
```

Comment:
Base of packet If this can help you [here](https://gist.github.com/DoctorKraft/78c3002b896ccf9f153aede043503617)